fastlane/spec/action_metadata_spec.rb

```diff
@@ -3,6 +3,10 @@
 describe Fastlane::Action do
   Fastlane::ActionsList.all_actions do |action, name|
     describe name do
+      before do
+        allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return("./fastlane")
```

Comment:
@mgrebenets 

This is required because of below behavior of `FastlaneCore::FastlaneFolder.path`, right?

https://github.com/fastlane/fastlane/blob/37fb600751bfc17d49da1d4f4369cee0adab771b/fastlane_core/lib/fastlane_core/fastlane_folder.rb#L12
```ruby
value = nil if Helper.is_test? # this is required, as the tests would use the ./fastlane folder otherwise
```

I think this behavior should be deleted. What do you think?
I have an idea, so I'll PR for this.
After that, I'd like to discuss more about the way we test at this part.